aiida/cmdline/commands/cmd_rabbitmq.py

```diff
@@ -0,0 +1,386 @@
+
+if t.TYPE_CHECKING:
+    import kiwipy.rmq
+
+    from aiida.manage.configuration.profile import Profile
+
+
+@verdi_devel.group('rabbitmq')
+def cmd_rabbitmq():
+    """Commands to interact with RabbitMQ."""
+
+
+@cmd_rabbitmq.group('queues')
+def cmd_queues():
+    """Commands to interact with RabbitMQ queues."""
+
+
+@cmd_rabbitmq.group('tasks')
+def cmd_tasks():
+    """Commands related to process tasks."""
+
+
+available_projectors = (
+    'arguments',
+    'auto_delete',
+    'backing_queue_status',
+    'consumer_utilisation',
+    'consumers',
+    'durable',
+    'effective_policy_definition',
+    'exclusive',
+    'exclusive_consumer_tag',
+    'garbage_collection',
+    'head_message_timestamp',
+    'idle_since',
+    'memory',
+    'message_bytes',
+    'message_bytes_paged_out',
+    'message_bytes_persistent',
+    'message_bytes_ram',
+    'message_bytes_ready',
+    'message_bytes_unacknowledged',
+    'messages',
+    'messages_details',
+    'messages_paged_out',
+    'messages_persistent',
+    'messages_ram',
+    'messages_ready',
+    'messages_ready_details',
+    'messages_ready_ram',
+    'messages_unacknowledged',
+    'messages_unacknowledged_details',
+    'messages_unacknowledged_ram',
+    'name',
+    'node',
+    'operator_policy',
+    'policy',
+    'recoverable_slaves',
+    'reductions',
+    'reductions_details',
+    'single_active_consumer_tag',
+    'state',
+    'type',
+    'vhost',
+)
+
+
+def echo_response(response: requests.Response, exit_on_error: bool = True) -> None:
+    """Echo the response of a request.
+
+    :param response: The response to the request.
+    :param exit_on_error: Boolean, if ``True``, call ``sys.exit`` with the status code of the response.
+    """
+    try:
+        response.raise_for_status()
+    except requests.HTTPError:
+        click.secho(f'[{response.status_code}] ', fg='red', bold=True, nl=False)
+        click.secho(f'{response.reason}: {response.url}')
+        if exit_on_error:
+            sys.exit(response.status_code)
+    else:
+        click.secho(f'[{response.status_code}] ', fg='green', bold=True)
+
+
+@wrapt.decorator
+@click.pass_context
+def with_client(ctx, wrapped, _, args, kwargs):
+    """Decorate a function injecting a :class:`aiida.manage.external.rmq.client.RabbitmqManagementClient`."""
+
+    from aiida.manage.external.rmq.client import RabbitmqManagementClient
+    config = ctx.obj.profile.process_control_config
+    client = RabbitmqManagementClient(
+        username=config['broker_username'],
+        password=config['broker_password'],
+        hostname=config['broker_host'],
+        virtual_host=config['broker_virtual_host'],
+    )
+
+    if not client.is_connected:
+        echo.echo_critical(
+            'Could not connect to the management API. Make sure RabbitMQ is running and the management plugin is '
+            'installed using `sudo rabbitmq-plugins enable rabbitmq_management`.'
```

Comment:
this worked nicely for me

Reply:
I'm assuming you ran this on a Ubuntu box? Not sure what our mileage will be for other ways of installing RabbitMQ or on other OS's

Reply:
As expected, @unkcpz experienced a problem because he is using RabbitMQ installed on docker. My suspicion is that it doesn't work because the 15672 port which is used by the API is not forwarded. I have updated the error message to include that information.

Reply:
yes, this was on ubuntu 20.04